dist/table-to-csv.js

```diff
@@ -36,6 +36,9 @@ angular.module('tv.tableToCsv', [])
 
         element.bind('click', function (e) {
           var table = e.target.previousElementSibling
+          var title = e.target.getAttribute('csv-title') || 'table-to-csv'
+          var downloadTime = new Date().toLocaleString('en-GB')
```

Comment:
I'd be tempted to remove this and just allow the user to pass it in as part of the attribute if they wish.